media/css/firefox/whatsnew/whatsnew-105-eu.scss

```diff
@@ -0,0 +1,161 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-pocket';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import 'includes/base';
+
+// * -------------------------------------------------------------------------- */
+// Main content
+
+.wnp-content-main {
+    background-color: #ffd25e;
+
+    .mzp-c-wordmark.mzp-t-product-pocket {
+        background: transparent url('/media/img/firefox/whatsnew/whatsnew105-eu/pocket-wordmark.svg') center center no-repeat;
+        width: 100%;
+    }
+
+    @media #{$mq-lg} {
+        .mzp-c-wordmark.mzp-t-product-pocket {
+            background-position: center left;
+        }
+
+        .mzp-c-split-body {
+            margin-bottom: $spacing-2xl;
+        }
+    }
+}
+
+.mzp-l-split-pop-top.mzp-c-split {
+    padding-top: 70px;
+}
+
+.wnp-main-title {
+    @include text-title-md;
+    color: #00256d;
+}
+
+.wnp-main-tagline {
+    @include text-body-lg;
+    color: #00256d;
+}
+
+.wnp-main-cta {
+    @include text-body-lg;
+    margin-top: $layout-md;
+
+    .mzp-c-button.mzp-t-product {
+        background-color: #00256d;
+
+        &:hover,
+        &:focus {
+            background-color: #000;
+        }
+    }
+}
+
+.t-pattern {
+    background: #ff8a00 url('/media/img/firefox/whatsnew/whatsnew105-eu/pattern.svg') center center no-repeat;
+    height: 114px;
+    width: 100%;
+}
+
+// * -------------------------------------------------------------------------- */
+// KOL variation (/de/firefox/105.0/whatsnew/?v=2)
+
+.wnp-header-kol {
+    background-color: #ffd25e;
+    padding-top: $spacing-2xl;
+    text-align: center;
+
+    .mzp-l-content {
+        padding-bottom: 0;
+        padding-top: 0;
+    }
+
+    .wnp-main-tagline {
+        @include text-body-xl;
+
+        br {
+            display: none;
+        }
+    }
+
+    .mzp-c-wordmark.mzp-t-product-pocket {
+        background: transparent url('/media/img/firefox/whatsnew/whatsnew105-eu/wordmark-lockup.svg') center center no-repeat;
```

Comment:
**issue (blocking)**: longer lockup wordmark is cut off on mobile

**suggestion**: since it's probably still too big when scaled down, might be best to use simple pocket logo until medium or larger screens

<img width="433" alt="Screen Shot 2022-09-12 at 2 01 17 PM" src="https://user-images.githubusercontent.com/19650432/189660652-b5cf62d6-2482-4add-a3e0-da5ad860fda7.png">

Reply:
Scales better than I thought - nice fix!